### PR TITLE
Show payments in pdf bill

### PR DESCRIPTION
**Pull request by joneskoo**
Works for me locally. It's late and this is pretty important piece of code. Please review carefully.

I renamed some variables, which is a likely place for mistakes, and didn't verify the bar code content yet.

[sikteeri_invoicess8hiX.pdf](https://github.com/kapsiry/sikteeri/files/310505/sikteeri_invoicess8hiX.pdf)
[sikteeri_reminder02HS2j.pdf](https://github.com/kapsiry/sikteeri/files/310506/sikteeri_reminder02HS2j.pdf)

ping @annttu @AriMartti

### CODE REVIEW

File: membership/management/commands/generate_test_pdf.py [hunk 1]
```diff
@@ -1,62 +1,86 @@
...
+from membership.models import Contact, Membership, BillingCycle, Payment
 
 
+@transaction.atomic
 def gen_test_PDFs():
-
-
-    contact = Contact()
-    contact.first_name = u"Antero"
-    contact.given_name = u"Antero Ilmari"
```
Comment (joneskoo) [inline membership/management/commands/generate_test_pdf.py]: given_name was actually a typo and nobody noticed…

File: membership/management/commands/generate_test_pdf.py [hunk 2]
```diff
@@ -1,62 +1,86 @@
...
-    member = Membership()
-    member.id = 4444
-    member.type = 'P'
-    member.status = 'A'
-    member.public_memberlist = True
-    member.municipality = 'Helsinki'
-    member.nationality = 'Suomi'
-    member.birth_year = 1990
-    member.person = contact
-    member.locked = False
```
Comment (joneskoo) [inline membership/management/commands/generate_test_pdf.py]: locked is actually a date and not boolean, nobody noticed.

File: membership/management/commands/generate_test_pdf.py [hunk 3]
```diff
@@ -1,62 +1,86 @@
...
+        billingcycle=cycle,
+        reference_number='1234',
+        transaction_id=uuid.uuid4(),
+        payment_day=datetime.now(),
+        amount=30.0,
+        payer_name="It was me")
+    reminder_file = tempfile.NamedTemporaryFile(
+        prefix="sikteeri_reminder", suffix=".pdf", delete=False)
+    reminder = pdf.PDFReminder(reminder_file)
+    reminder.addCycle(cycle, payments=MockPayments)
```
Comment (joneskoo) [inline membership/management/commands/generate_test_pdf.py]: Actually in hindsight the mock payments is probably redundant now that I ended up implementing the database rollback. Should use model instead :)

**Comment by AriMartti** [comment]
Verified the barcode in attached reminder, passed my eyesight validation.

**Comment by annttu** [comment]
Looks good!

**Pull request merged**